manim/animation/creation.py

```diff
@@ -153,8 +156,22 @@ def _get_bounds(self, alpha: float) -> typing.Tuple[int, float]:
         return (0, alpha)
 
 
-class Uncreate(ShowCreation):
-    """Like :class:`ShowCreation` but in reverse.
+class ShowCreation(Create):
+    """Deprecated. Use :class:`~.Create` instead."""
+
+    logger.warning(
+        "ShowCreation has been deprecated in favor of Create. Please use Create instead!"
+    )
+
+    def __init__(self, mobject: VMobject, lag_ratio: float = 1.0, **kwargs) -> None:
+        super().__init__(mobject, lag_ratio=lag_ratio, **kwargs)
```

Comment:
Move logger.warning() into the `__init__`

Reply:
something like this
```suggestion
    def __init__(self, mobject: VMobject, lag_ratio: float = 1.0, **kwargs) -> None:
        logger.warning(
            "ShowCreation has been deprecated in favor of Create. Please use Create instead!"
        )
        super().__init__(mobject, lag_ratio=lag_ratio, **kwargs)
```

Reply:
Just traced the issue to this after looking for other deprecations are carried out and saw your reply about 2 minutes after getting it fixed! Feeling both proud and stupid at the same time :D
Thank you!